04-type-system/01-construct-duration/.README.md

```diff
@@ -0,0 +1,3 @@
+# Construct duration from arguments
+
+Write a small function that accepts two arguments ({{index . "arg1"}}, {{index . "arg2"}}), construct a time instant based on them, where {{index . "arg1"}} is the number of {{index . "type1"}}, while {{index . "arg2"}} is the number of {{index . "type2"}}. The function returns the result as `time.Duration`. Place your code in the `exercise.go` file near tho the placeholder `// INSERT YOUR CODE HERE`
```

Comment:
The result should be the sum of the two? Can you please add an example to the README?